src/api/mobility.ts

```diff
@@ -40,7 +41,7 @@ import {
 
 type VehicleRequestOpts = Pick<AxiosRequestConfig, 'signal'>;
 
-export const getVehicles = (
+export const getVehicles = async (
```

Comment:
I think it doesn't make sense to add `async` for functions that doesn't use `await`…